Use updated version of node-sqlite3 so that python3 can be used to
build it.
(https://github.com/mapbox/node-sqlite3/commit/3fb3715c4e699ca3bc19e47b136758c66f61e477)

This fork is a one character change from master in deps/sqlite3.gyp,
explicity specifying python3:
-          'action': ['<!(node -p "process.env.npm_config_python || \\"python\\"")','./extract.py','./sqlite-autoconf-<@(sqlite_version).tar.gz','<(SHARED_INTERMEDIATE_DIR)']
+          'action': ['<!(node -p "process.env.npm_config_python || \\"python3\\"")','./extract.py','./sqlite-autoconf-<@(sqlite_version).tar.gz','<(SHARED_INTERMEDIATE_DIR)']
--- thelounge-4.2.0/package.json
+++ thelounge-4.2.0/package.json
@@ -68,7 +68,7 @@
     "yarn": "1.22.4"
   },
   "optionalDependencies": {
-    "sqlite3": "5.0.0"
+    "sqlite3": "git+https://github.com/KevinThomas0/node-sqlite3.git#fca87deae32297c3d1b67391937a8e6883f2f802"
   },
   "devDependencies": {
     "@babel/core": "7.11.1",
